src/Microsoft.Framework.ApplicationHost2/Program.cs

```diff
@@ -0,0 +1,286 @@
+// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using System.Threading.Tasks;
+using Microsoft.Framework.ApplicationHost.Impl.Syntax;
+using Microsoft.Framework.Runtime;
+using Microsoft.Framework.Runtime.Common;
+using Microsoft.Framework.Runtime.Common.CommandLine;
+
+namespace Microsoft.Framework.ApplicationHost
+{
+    public class Program
+    {
+        private readonly IAssemblyLoaderContainer _container;
+        private readonly IApplicationEnvironment _environment;
+        private readonly IServiceProvider _serviceProvider;
+
+        public Program(IAssemblyLoaderContainer container, IApplicationEnvironment environment, IServiceProvider serviceProvider)
+        {
+            _container = container;
+            _environment = environment;
+            _serviceProvider = serviceProvider;
+        }
+
+        public Task<int> Main(string[] args)
+        {
+            Console.WriteLine("I AM APP HOST 2!");
+            Console.WriteLine("TODO: Actually do things!");
+            return Task.FromResult(0);
+
+            //DefaultHostOptions options;
```

Comment:
Removing all these commented codes? They can be copied from the existing code base anyway.